### PR TITLE
Transformer

### PR DESCRIPTION
**Pull request by chloealee**
#### What's the PR do?

Adds the drupal_id to the `Transformer.php`
#### Where should the reviewer start?

`Transformer.php` line 461
#### How should this be manually tested?

Hit `/reportback-items.json` endpoint to view the exposed `drupal_id` in the user object.  
#### What are the relevant tickets?

Meant to put in this fix #5100

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -37,16 +37,16 @@ Vagrant.configure("2") do |config|
...
   # With Varnish
-  config.vm.network :forwarded_port, guest: 6081, host: 9999
+  config.vm.network :forwarded_port, guest: 6081, host: 3333
 
   # Solr.
-  config.vm.network :forwarded_port, guest: 8983, host: 8983
+  config.vm.network :forwarded_port, guest: 8983, host: 4444
 
   config.vm.provision :shell, :inline => 'more /vagrant/scripts/install_complete.txt'
 end
```
Comment (weerd) [inline Vagrantfile:52]: Should this Vagrant file be committed and updated?

File: lib/modules/dosomething/dosomething_api/includes/Transformer.php [hunk 2]
```diff
@@ -458,6 +458,7 @@ protected function transformReportbackItem($data) {
   protected function transformUser($data) {
     return array(
       'id' => $data->id,
+      'drupal_id' => $data->drupal_id,
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_api/includes/Transformer.php:461]: So I don't think we actually want to return the Drupal ID back in the API response. Drupal ID shouldn't mean anything to anyone using the api... is there a specific reason why we need this returned?

If we **do** need it, we could call it something agnostic to the framework, like `legacy_id` or something. But ideally we shouldn't be outputting this back in the JSON API Response...

Comment (chloealee) [inline lib/modules/dosomething/dosomething_api/includes/Transformer.php:461]: Oooh okay, my mistake! I thought we wanted this but totally makes sense! Should I just close out this PR?

**Comment by weerd** [comment]
If we don't need the Vagrant update either, then yeah just close it on up :)

**Pull request closed without merging**